### ISSUE TITLE
Make sure aliases can set all/most build/run flags

### ISSUE THREAD

**Issue by quintesse** [jbangdev/jbang#1486]
Right now `alias add` can't set flags like `--offline` and `--fresh`. We should take al ook at which flags make sense to enable for aliases.

**Comment by maxandersen** [comment]
Yes, but would t the proper solution for out of date catalogs be that they refresh if stale?

**Comment by quintesse** [comment]
No, this is not for the catalog, it's just that if someone wants to create an alias to run `jbang run/build --fresh` or `jbang run/build --offline` it would be nice if that were possible. Right now it isn't.

**Comment by maxandersen** [comment]
I meant the right solution for the orignal discussion at https://github.com/jbangdev/jbang/issues/1485

**Comment by quintesse** [comment]
Perhaps I misunderstood that issue, but I thought it to mean that it didn't automatically update to the latest version of the app. I assumed there's nothing in the alias itself that changed. (Eg. the alias changes from running `foo1@bar` to `foo2@bar`).
